src/MedTrackerAPI/Features/Supplies/AddSupply.cs

```diff
@@ -0,0 +1,65 @@
+using FluentValidation;
+using MediatR;
+using MedTrackerAPI.Endpoints;
+using MedTrackerAPI.Infrastructure;
+
+namespace MedTrackerAPI.Features.Supplies;
+
+public static class AddSupply
+{
+    public record Response (int SupplyId, string? Description, string? Manufacturer, string? PartNumber, string? LotNumber, int DeviceId);
+    
+    public class AddSupplyCommandValidator : AbstractValidator<AddSupplyCommand>
+    {
+        public AddSupplyCommandValidator()
+        {
+            RuleFor(x => x.Description).NotEmpty().WithMessage("Description is required.");
```

Comment:
The validation errors come OOTB with a message very similar to this. You can keep this or just ride the library.